packages/grafana-ui/src/options/builder/tooltip.tsx

```diff
@@ -1,21 +1,40 @@
 import { PanelOptionsEditorBuilder } from '@grafana/data';
 import { OptionsWithTooltip, TooltipDisplayMode, SortOrder } from '@grafana/schema';
 
+export enum TooltipModeOptions {
```

Comment:
`TooltipModes`?

Reply:
i think enums tend to be singular

![image](https://user-images.githubusercontent.com/43234/211496400-5e36a294-d51f-4e10-a1d2-ee5ae00108b6.png)